cms/core/tests/utils.py

```diff
@@ -33,3 +38,16 @@ def extract_datalayer_pushed_values(html_content: str) -> dict[str, object]:
         arg_values = json.loads(datalayer_arg)
         datalayer_values.update(arg_values)
     return datalayer_values
+
+
+class PanelBlockAssertions(SimpleTestCase):
+    """Mixin that provides shared assertions for panel blocks."""
+
+    def assertPanelBlockFields(
```

Comment:
This is a pre-existing issue as not all methods and functions have a return type, but we could add `-> None` here. Non-blocking though.

(I think mypy skips test files?)